tests/conftest.py

```diff
@@ -190,8 +190,15 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
                 println("Removing missing tests from record.")
             println("")
 
+    if ui_option == "test" and _should_write_ui_report(exitstatus):
+        println("\n-------- Suggested fixtures.json diff: --------")
+        ui_tests.print_actual_fixtures(config.getoption("ui_check_missing"))
+        println("")
```

Comment:
Maybe I should put this block in front of the "List of missing tests" so it is not barried under the long json output?

Reply:
i don't think that's necessary? if the JSON list is long, you'll need to scroll up anyway and see the missing ones

also won't the missing ones be included in the json diff?

Reply:
If I run `--ui=test --ui-check-missing` and a fixture is missing in fixtures.json it is not printed in the diff. Which is correct I guess? I am wondering if it makes even sense to print the suggested diff in case the test suite fails on "missing"?

Reply:
why not? you can also directly add the lines to the JSON to fix the "missing" cases, no?

Reply:
I have to admit I am a bit confused :neutral_face:. `--ui-check-missing` checks whether something is superfluous in the fixtures.json or in another words if some test that was expected did not run. That means it is a [difference](https://github.com/trezor/trezor-firmware/blob/f0e3c592e7adb67c8130077a163f6f25f8099416/tests/ui_tests/__init__.py#L108-L107) between what is contained in the fixtures and what is being tested. For example if we have A in fixtures.json but A is not run, A is printed as missing. Which we maybe wanted to have the other way around?